actions.yaml

```diff
@@ -4,3 +4,6 @@
 get-admin-password:
   description: |
     Retrieve the admin password which has been auto-generated during Jenkins install.
+rotate-credentials:
+  description: |
+    Invalidate all current sessions and generate a new password for admin account.
```

Comment:
Out of curiosity: why there is one action to do two things instead of two separate actions like invalidate-all-sessions and generate-admin-password ?

Reply:
This is taken from an action in wordpress as well, where this action is used to re-secure the application from a data breach - invalidate all the logged in users and rotate the password. So for example, when the admin user is breached, unless we log out that user, the admin user will still be able to cause damage while the session lasts.

Reply:
I'm not familiar with Jenkins so I was wondering if this is usually needed this way instead of creating two actions :-)